server/service/client_software.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // ListSoftware retrieves the software running across hosts.
 func (c *Client) ListSoftware(query string) ([]fleet.Software, error) {
-	verb, path := "GET", "/api/latest/fleet/software"
+	verb, path := "GET", "/api/latest/fleet/software/versions"
 	var responseBody listSoftwareResponse
```

Comment:
Not critical because I will adjust as necessary for the CLI ticket, but I think the response would now come from `listSoftwareVersionsResponse`.